File: Selenium/Activities/Activity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Firefox() as driver:
    
    driver.get("https://training-support.net/")

    
    print("Page title is: ", driver.title)

   
    driver.find_element(By.LINK_TEXT, "About Us").click()

    
    print("New page title is: ", driver.title)

File: Selenium/Activities/Activity1.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {

	public static void main(String[] args) {
WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net");
		System.out.println(driver.getTitle());
		
	    driver.findElement(By.linkText("About Us")).click();
	    System.out.printf("About us page",driver.getTitle());
		driver.close();

		
	}

}
